resolwe_bio/kb/filters.py

```diff
@@ -5,11 +5,114 @@
 =======
 
 """
+from functools import reduce
+
 import django_filters as filters
+from django.contrib.postgres.search import SearchQuery, SearchRank
+from django.db import models
+from django.db.models import F
 
 from resolwe.flow.filters import TEXT_LOOKUPS
 
-from .models import Mapping
+from .models import Feature, Mapping
+
+
+class FullTextFilter(filters.BaseCSVFilter):
+    """Filter for full-text search."""
+
+    def filter(self, queryset, value):
+        """Filter field by text-search vector.
+
+        Values, given in csv format are joined with the OR operator.
+
```

Comment:
Maybe consider renaming second argument from value to values?

Reply:
Done.